src/common/HaskellDo/Toolbar/State.hs

```diff
@@ -134,7 +137,26 @@ update ToggleError state = do
     localIO toggleError
     return state
 
+update ConvertToPDF state = do
+    checkIfInstalled <- atRemote . localIO $ readProcess "find" ["/usr/bin","-name","wkhtmltopdf"] []
+    (errorCode,_,_) <- atRemote . localIO $ readCreateProcessWithExitCode (shell "which wkhtmltopdf") ""
```

Comment:
It looks like you're checking the existence of the `wkhtmltopdf` binary twice here. I'd use `which`, that not only tells you if the file exists or not, but also what's its location. I guess there will be people out there that have installed `wkhtmltopdf` somewhere else, so for them, I think that this code will fail when it tries to find the binary in `/usr/bin`